www/index.md

```diff
@@ -2,17 +2,14 @@
 layout: page
 ---
 
-
 ### Getting Started
 
 - [Quick Start](quick-start.html) - the best place to start reading if you are new to Mergin
```

Comment:
quick start has a couple of outdated screenshots (maybe text as well?)

Reply:
Let me handle that in another PR:
https://github.com/lutraconsulting/mergin-docs/issues/23